src/helpers.js

```diff
@@ -11,7 +10,7 @@ export function isOAS3(spec) {
     return false;
   }
 
-  return startsWith(oasVersion, '3');
+  return oasVersion[0] === '3';
```

Comment:
wouldn't following form be more readable in this case?

```js
return oasVersion.charAt(0) === '3';
```

By looking at above `charAt` I guess I'd still prefer a `startsWith` method here:

```js
return oasVersion.startsWith('3')
```
It's declarative and it does exactly what it says. What's your take on this?

Reply:
They all solves the issue. I Just took the most easiest, low level, fastest, and shortest method available. 

I don't really care about which of them are being used.
charAt has better support then newer startswith

My method could work for both string and array of it where splited

Reply:
This library minimum required versions of Node.js is now at 12.4. So as long as we use features that falls withing this minimum required version we're fine. 

> I don't really care about which of them are being used.

If you don't have preference let use `startsWith`. It increases the readability a bit.

Reply:
fixed